Formula/alot.rb

```diff
@@ -43,54 +53,59 @@ class Alot < Formula
     sha256 "586372eb92059873e29eba4f9dec8381541b4d3834660707faf8ba59146dfc35"
   end
 
+  resource "gpg" do
+    url "https://files.pythonhosted.org/packages/ef/86/c5a34243a932346c59cb25eb49a4d1dec227974209eb9b618d0ed57ea5be/gpg-1.10.0.tar.gz"
+    sha256 "349214a866c84aa548bc35ed14eccd2ec9085b3958d5753a63a19a71a1f523ca"
+  end
```

Comment:
This is provided by `depends_on "gpgme"` I think, and the formula has a newer version.

Reply:
👍🏻 I used the same tool we used to use last time I was involved in Homebrew so I suspect my dependency update methodology here was a little outdated. I'll change this.

Reply:
We have a `brew update-python-resources` now! Might need to adjust `pypi_formula_mappings.json` though to give `brew update-python-resources` a hint that it should skip it.

Reply:
I tweaked `pypi_formula_mappings.json` in a way that sort of made sense to me after I spotted it but you may need to nudge me on what needs changing further if that wasn't sufficient.

Reply:
Ths JSON file is solely for `brew update-python-resources` so it's just a case of tweaking it if the result of that command is not what you expect.

I just realised however that `alot` isn't on PyPI so my suggestion perhaps won't work, in which case you don't need to touch the JSON at all. I'm surprised `homebrew-pypi-poet` works (assuming that's what you were using).

Reply:
>  it's just a case of tweaking it if the list of resources from that command is not what you expect.

I suspect it'd leave `wheel` out as non-essential. I noticed from the `json` file that's a bit of a theme throughout.

> I'm surprised homebrew-pypi-poet works in that case (assuming that's what you were using).

It was indeed!

```
cd /tmp
mkdir venv && cd venv
bin/pip install https://github.com/pazz/alot/archive/refs/tags/0.9.1.tar.gz
bin/pip install homebrew-pypi-poet
bin/poet alot
```

Reply:
I suppose it's a slight shortcoming of `brew update-python-resources` then. It doesn't download/install anything and just relies on PyPI searching - which won't work if it's not on PyPI!

> I suspect it'd leave `wheel` out as non-essential. I noticed from the `json` file that's a bit of a theme throughout.

Yes, there was an issue where this was the case. Think it was fixed recently in https://github.com/Homebrew/brew/pull/10849.